swagger_spec_compatibility/walkers/additional_properties.py

```diff
@@ -39,6 +40,45 @@ def fix_parameter_path(self, path, original_path):
         )
 
 
+CycleSentinel = collections.namedtuple('CycleSentinel', ('path',))
```

Comment:
I would recommend to use `typing.NamedTuple` such that the sentinel object is properly typed

```suggestion
from swagger_spec_compatibility.walkers import PathType

CycleSentinel = typing.NamedTuple(
    'CycleSentinel',
    (
            ('path', PathType),
    )
)
```

Reply:
I didn't think this would work since this library still has to support Python 2.7, but let me try it given we're installing the backport.

Reply:
It is already in the lib :P

Reply:
This is already present in the codebase

https://github.com/Yelp/swagger-spec-compatibility/blob/325e5280f5a936ec9d1b7646fed914846d3170f7/swagger_spec_compatibility/spec_utils.py#L129-L134